data/124/276/627/9/1242766279.geojson

```diff
@@ -28,21 +28,27 @@
     "lbl:max_zoom":18.0,
     "lbl:min_zoom":13.5,
     "mz:hierarchy_label":1,
-    "mz:is_current":-1,
+    "mz:is_current":1,
     "mz:min_zoom":12.0,
+    "name:eng_x_preferred":[
+        "Atawala"
+    ],
+    "name:eng_x_variant":[
+        "Etaola"
+    ],
     "src:geom":"geonames",
```

Comment:
Nit: Should this source be `wof` now since you've modified the geom lat/lng?

Reply:
Hmm I don't think the geometry of this feature was updated? I think the diff in the geometry is due to the exportify tool..